newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/index.js

```diff
@@ -31,26 +31,37 @@ import FlatButton from '../../../../UI/FlatButton';
 import Add from '../../../../UI/CustomSvgIcons/Add';
 import TeamMemberProjectsView from './TeamMemberProjectsView';
 import Refresh from '../../../../UI/CustomSvgIcons/Refresh';
-import { ColumnStackLayout } from '../../../../UI/Layout';
+import { ColumnStackLayout, LineStackLayout } from '../../../../UI/Layout';
 import Paper from '../../../../UI/Paper';
 import { useResponsiveWindowSize } from '../../../../UI/Responsive/ResponsiveWindowMeasurer';
 import RaisedButton from '../../../../UI/RaisedButton';
-import { groupMembersByGroupId } from './utils';
+import { groupMembersByGroupId, sortGroupsWithMembers } from './Utils';
 import ErrorBoundary from '../../../../UI/ErrorBoundary';
 import ContextMenu, {
   type ContextMenuInterface,
 } from '../../../../UI/Menu/ContextMenu';
 import type { ClientCoordinates } from '../../../../Utils/UseLongTouch';
 import { type MenuItemTemplate } from '../../../../UI/Menu/Menu.flow';
 import { EducationCard } from '../LearnSection/EducationCard';
+import UserSVG from '../../../../UI/CustomSvgIcons/User';
+import { copyTextToClipboard } from '../../../../Utils/Clipboard';
+import ManageEducationAccountDialog from './ManageEducationAccountDialog';
+import TeamAvailableSeats from './TeamAvailableSeats';
+import StudentCreationCard from './StudentCreationCard';
 
 const PADDING = 16;
 
 const styles = {
   list: { padding: 0 },
   lobbyContainer: { padding: PADDING },
   roomsContainer: { paddingRight: PADDING },
-  titleAdornmentContainer: { paddingRight: PADDING },
+  manageSeatsInsertContainer: {
+    padding: 16,
+    display: 'flex',
+    alignItems: 'center',
+    border: '1px solid white',
```

Comment:
What about `gdevelopTheme.palette.secondary`? Hardcoding a color seems risky.

Check CalloutCard.js

Reply:
It's overriden in the JSX, but I removed the color from this style just in case.